odoo/addons/base/models/assetsbundle.py

```diff
@@ -693,10 +791,25 @@ def minify(self):
     def with_header(self, content=None):
         if content is None:
             content = self.content
-        return '\n/* %s */\n%s' % (self.name, content)
+        return f'\n/* {self.name} */\n{content}'
 
 
 class JavascriptAsset(WebAsset):
+
+    def __init__(self, bundle, inline=None, url=None, filename=None):
+        super().__init__(bundle, inline, url, filename)
+        self.is_odoo_module = is_odoo_module(super().content)
```

Comment:
Note that the attribute and the function have different semantics:
* the attribute tells whether the content is transpiled;
* the function determines whether the content is an Odoo module.

Using names corresponding to their semantics would clarify the code:
```suggestion
        self.is_transpiled = is_odoo_module(super().content)
```